_data/clients/weechat.yml

```diff
@@ -0,0 +1,5 @@
+name: Weechat
```

Comment:
```suggestion
name: WeeChat
```

I wonder if it should say "WeeChat XMPP plugin" or something as WeeChat doesn't support XMPP out of the box.

Reply:
Like so?